### PR TITLE
fix: Placeholder values of author name and example is not showing now in GroupWise Skills

### LINKED ISSUE #2071: GroupWise skills showing placeholder values of example and author name
**Describe the bug**
GroupWise skills fragment is showing placeholder values for author name and example, i.e. "<author_name>" for an author name and "\<The question that should be shown in public skill displays\>" for an example.

**Expected Behaviour**
Author name should be set to --> "Author not found"
The example should be set to --> ""

**Steps to reproduce it**
Click on any category of skills and try to search for skills with author name as <author_name> and example as "\<The question that should be shown in public skill displays\>"

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by naman653**
Fixes #2071 

Changes: Now placeholder values of example and author name of skill data is not showing

### CODE REVIEW

**Comment by naman653** [comment]
It was just checking if the author name or example is empty or not. But if there was default placeholder value present, it sets these values to those default ones showing author name as <author_name> and same as for example.

> Why was it showing the placeholder values?

**Comment by iamareebjamal** [comment]
But from where were the placeholder values coming from?

**Comment by naman653** [comment]
These values are directly coming from the server along with skills data. There is a template to create a new skill, and if the user doesn't replace placeholder values while creating skills, then these values are set in as data.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -438,6 +438,10 @@
     <string name="report_send_success">Skill reported successfully</string>
     <string name="reported_already">Skill already reported</string>
 
+    <!--Default placeholder values of skills description-->
+    <string name="default_author_name" translatable="false">&lt;author_name&gt;</string>
+    <string name="default_example" translatable="false">&lt;The question that should be shown in public skill displays&gt;</string>
```
Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: Remove from strings and make String constants

Comment (naman653) [reply app/src/main/res/values/strings.xml]: I thought of that, but the same problem exists in the skill details fragment. So I thought these strings might be useful in the future. Please enlighten, if asked changes should be done or not.

Comment (iamareebjamal) [reply app/src/main/res/values/strings.xml]: They should be done

**Pull request merged**